Src/FluentAssertions/Common/MethodInfoExtensions.cs

```diff
@@ -3,27 +3,64 @@
 using System.Linq;
 using System.Linq.Expressions;
 using System.Reflection;
+using System.Runtime.CompilerServices;
 
 namespace FluentAssertions.Common
 {
     public static class MethodInfoExtensions
     {
+        private static Lazy<int> ImplementationOptionsMask = new Lazy<int>(() => Enum.GetValues(typeof(MethodImplOptions)).Cast<int>().Sum(x => x));
```

Comment:
Why lazy and not just `static readonly`?

Reply:
I wanted to avoid computing that if it won't be used in a check. And if it were static it would have computed every time somebody uses the assembly. It's probably a nanosecond win anyway ;).